addons/l10n_generic_coa/migrations/12.0.1.1/noupdate_changes.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version='1.0' encoding='utf-8'?>
+<odoo>
+  <record id="conf_cas_interim1" model="account.account.template">
+    <field name="reconcile" eval="True"/>
+  </record>
+  <record id="conf_cas_interim2" model="account.account.template">
+    <field name="reconcile" eval="True"/>
+  </record>
+  <record id="configurable_chart_template" model="account.chart.template">
+    <!--<field name="transfer_account_id"/>-->
+    <field name="transfer_account_code_prefix">1017</field>
```

Comment:
Shouldn't this value be calculated from the `openupgrade_legacy_12_0_transfer_account_id`? It is possible that the old `transfer_account_id` (defined as `noupdate="1"`) has been manually modified.

Reply:
Right. In fact, it was handled in the `fill_account_chart_template_account_code_prefix` method of post-migration of `account`.